packages/components/src/Actions/ActionFile/ActionFile.component.js

```diff
@@ -133,9 +133,9 @@ class ActionFile extends React.Component {
 					name={name}
 					id={localId}
 					disabled={inProgress || disabled}
-					className={theme['action-file-label']}
+					className={classNames(theme['action-file-input'], 'sr-only')}
```

Comment:
I'm not sure, if you have talk about the class 'sr-only' during the previous meeting but that can be interested to talk on it during a next meeting or in a doc (with a reference to the bootstrap documentation for exemple)

Reply:
Good suggestion.